### DISCUSSION TITLE
pstbsserver loads the wrong queryport

### DISCUSSION THREAD

**Discussion by DerTonka** [GameServerManagers/LinuxGSM#3595] [Q&A] [answered]
Current version (version="v21.3.1") from pstbsserver loads constantly wrong queryport. The port is set correct to 10037 but the output is incorrect

`grep -r "10037" lgsm/
lgsm/config-lgsm/pstbsserver/_default.cfg:queryport="10037"
lgsm/config-default/config-lgsm/pstbsserver/_default.cfg:queryport="10037"`

DESCRIPTION  PORT   PROTOCOL  LISTEN
Game                 10027  udp       1
Query                10027  udp       1
RCON                 21114  tcp       1

`./PostScriptumServer.sh MULTIHOME=0.0.0.0 RANDOM=NONE Port=10027 QueryPort=10027 RCONPORT=21114 FIXEDMAXPLAYERS=79 NumReservedSlots=0
`

Settings in pstbsserver.cfg:

ip="0.0.0.0"
port="10027"
queryport="10037"
rconport="21114"

Output
DESCRIPTION  PORT   PROTOCOL  LISTEN
Game                 10027  udp       0
Query                10027  udp       0
RCON                 21114  tcp       0

No problems with older version v21.2.5 tho.

**Comment by johnoclockdk** [comment, marked as answer]
this will be fix soon 

#3597
